### ISSUE TITLE
start server with errors

### ISSUE THREAD

**Issue by jiangyh1024** [pugjs/babel-plugin-transform-react-pug#129]
![image](https://user-images.githubusercontent.com/34930870/101878663-e963da80-3bca-11eb-86a2-9501ad714431.png)

not sure why

**Comment by ezhlobo** [comment]
@jiangyh1024 yeah, I've just upgraded to the latest react-scripts and got the same issue. When I added `DISABLE_NEW_JSX_TRANSFORM=true` it started working for a while but then died again with absolutely the same issue.

I'm investigating right now

**Comment by ezhlobo** [comment]
JFYI, I reproduced the issue after upgrading all the dependencies.

Assuming we need to drop node 6 and 8 support and upgrade major version of this plugin, I believe this fix should be coupled with that release, thus it's fine to upgrade internal dependencies here.

**Comment by ezhlobo** [comment]
Well, it turned out to be an issue in babel itself. Look at the issue and related PR: https://github.com/babel/babel/issues/12478

I think we should wait for new patch version of babel plugins, but in the meantime you can just specify older version of `@babel/plugin-transform-react-jsx`.

This is what I did for my project:
```json
"devDependencies": {
  "@babel/plugin-transform-react-jsx": ">= 7.12 < 7.12.10 || > 7.12.10"
}
```

CC @jiangyh1024

**Comment by jiangyh1024** [comment]
Thanks , appreciate it very much

发自我的iPhone

**Comment by skotchpine** [comment]
the babel issue looks resolved now
